datahub-web-react/src/app/ingest/source/builder/RecipeForm/utils.tsx

```diff
@@ -266,9 +355,11 @@ export const UPSTREAM_LINEAGE_IN_REPORT: RecipeField = {
     rules: null,
 };
 
-const TableLineageModeTooltip = () => {
-    return (
-        <div>
+export const TABLE_LINEAGE_MODE: RecipeField = {
+    name: 'table_lineage_mode',
+    label: 'Table Lineage Mode',
+    tooltip: (
+        <TableLineageModeTooltip>
```

Comment:
since `TableLineageModeTooltip` just wraps its children in a div, you could simplify this by simply using a `div` here instead of `TableLineageModeTooltip` and you can delete the extra component from above

Reply:
Also since this is wrapped in a `p` tag I don't think we need any other outside wrappers so you should be fine to remove `TableLineageModeTooltip` entirely

Reply:
Checked & Fixed